tests/component/test_channel_properties.py

```diff
@@ -1,11 +1,74 @@
 """Contains a collection of pytest tests that validates channel properties."""
 
+import pytest
+
+import nidaqmx
 from nidaqmx._task_modules.channels.ai_channel import AIChannel
 from nidaqmx._task_modules.channels.ci_channel import CIChannel
-from nidaqmx.constants import PowerIdleOutputBehavior
-
-
-def test__channel__get_boolean_property__returns_value(ai_voltage_chan_with_excit: AIChannel):
+from nidaqmx.constants import ExcitationSource, PowerIdleOutputBehavior, RTDType
+
+
+@pytest.fixture(scope="module")
```

Comment:
Scope=module reuses the same task for multiple test cases. This may make test failures harder to debug. For example, if you added an error test that sets an invalid value and verifies the task, then other test cases would raise an error when they verify the task.

Reply:
[Scope function added to fixture to destroy task at end of test](https://github.com/ni/nidaqmx-python/pull/241/commits/672297e0087f313b262de4a85ea76663e8fd3921)

Reply:
FYI, `scope='function'` is the default.

https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.fixture